src/main/java/net/minecraftforge/coremod/transformer/CoreModBaseTransformer.java

```diff
@@ -50,18 +69,35 @@ public T transform(T input, ITransformerVotingContext context) {
 
     abstract T runCoremod(T input);
 
-    @NotNull
+    /**
+     * The transformer vote that this CoreMod should use as a result of transformation.
+     *
+     * @param context The context of the vote
+     * @return The desired transformer vote
+     */
     @Override
     public TransformerVoteResult castVote(ITransformerVotingContext context) {
         return TransformerVoteResult.YES;
```

Comment:
We can look into behavioral changes for CoreMods 5.3 or 6.0.

Reply:
This should not be needed, as the transformers are gathered by class name.
Tho feature wise would be exposing this voting to the coremod js file itself. So that it can detect conflicting states.
But thats outside the scope of this PR.